coding_conventions.md

```diff
@@ -231,3 +231,57 @@ const ChargeModuleTokenService = require('../../app/services/charge-module-token
 
 // Start testing!
 ```
+## JSDoc Comments
+In node the standard way to document code is by using comments called [JSDoc]( https://jsdoc.app/) comments.
+ This style of comments has its own conventions that we follow.
+
+Comments begin with a slash-star-star (`/**`) and end with a star-slash (`*/`).
+
+Begin the JSDoc comment with a description of the code element, providing a clear and concise explanation of its purpose, behaviour, or usage.
+
+Tags: Use tags to provide additional information about the code element. Commonly used tags include `@param`, `@returns`, `@throws`, `@type`, `@example`, and `@see`.
+
+Specify the types of parameters, return values, and variables using curly braces {}. We use built-in JavaScript types (string, number, boolean, etc.) For arrays we notate this using square brackets `[]`
+
+For our JSDoc comments we don't end the sentence with a full stop (unless we include an expanded description). We also put spaces in between our descriptions.
+
+```javascript
+// Good comment
+/**
+ * Short description of the code element
+ *
+ * Expanded description, notes or examples if necessary.
+ *
+ * @param {String} name - Description of the params
+ *
+ * @returns {String} Description of what is returned
+ */
+
+// Bad comment
+/**
+ * Description of the code element.
+ * @param {String} name Description of the params.
+ * @returns {String} Description of what is returned.
+ */
+
+```
```

Comment:
I'd suggest using a 'real' document comment to serve as an example of what we are after. I think a bad comment can be inferred as anything that does not match it.

```suggestion
The following is an example of a 'good' method document comment.

```javascript
/**
 * Fetch all SROC charge versions linked to licences flagged for supplementary billing
 * 
 * This is used to get the charge versions that will be used to generate the bills for a given billing period. Excluding
 * those with a `draft` status, it selects all `charge_version` records that have a start date before the billing period
 * end date, and an end date that is either null or after the billing period's start date.
 * 
 * They must also be linked to licences in the specified region flagged for SROC supplementary billing. The results
 * include linked models that will be needed in the billing process, for example, a charge version's `charge_elements`.
 *
 * @param {String} regionId - UUID of the region being billed that the licences must be linked to
 * @param {Object} billingPeriod - Object with a `startDate` and `endDate` property representing the period being billed
 *
 * @returns {Promise<Object[]>} An array of matching charge versions
 */
async function go (regionId, billingPeriod) {
```
```